src/Microsoft.AspNetCore.Mvc.Core/Internal/DefaultAssemblyPartDiscoveryProvider.cs

```diff
@@ -26,7 +26,7 @@ public static class DefaultAssemblyPartDiscoveryProvider
             "microsoft.aspnetcore.mvc.formatters.xml",
             "microsoft.aspnetcore.mvc.localization",
             "microsoft.aspnetcore.mvc.razor",
-            "microsoft.aspnetcore.mvc.razor.host",
+            "microsoft.aspnetcore.mvc.razor.extensions",
```

Comment:
We shouldn't have `Razor.Runtime` here. This is specifically looking for MVC controllers.